### DISCUSSION TITLE
既然 dangerouslyApiKey 不安全，为什么还给出这种用法？有没有替代的安全使用方式？

### DISCUSSION THREAD

**Discussion by mainytht** [ant-design/x#243] [General]
这个项目非常好，就是有以上困惑...

**Comment by YumoImer** [comment]
命名为 `dangerouslyApiKey ` 是为了更好的提醒开发者使用这个属性可能带来安全隐患。

我想到的一些方式：

1. 使用代理服务，将 ApiKey 存储在服务器端
2. 如果必须在前端使用密钥，可以考虑对 ApiKey 加密

**Reply by YumoImer** [reply]
该问题已更新在官网。

https://x.ant.design/docs/react/dangerously-api-key
